serving/deploying-with-private-registry.md

```diff
@@ -0,0 +1,143 @@
+# Deploying to Knative using a private container registry
+This guide walks you through deploying an application to Knative from source code in a git repository using a private container registry for the container image. The source code should contain a dockerfile. For this guide, we'll use this [simple app](https://github.com/mchmarny/simple-app), but you could use your own.
```

Comment:
I think we should build one of the sample apps from within the Knative project. For example, this one uses one the Hello World sample in Go: https://github.com/knative/docs/blob/master/install/getting-started-knative-app.md

Reply:
Agree! We should considering doing this for some of the other samples that use external apps as well.